### PR TITLE
cifar10 initial commit

### PR DESCRIPTION
**Pull request by czabo**
CIFAR-10 dataset. Didn't add the tagging since there are no vision related tags.

### CODE REVIEW

File: datasets/cifar10/cifar10.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        )
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "img": datasets.Array3D(shape=(32, 32, 3), dtype="uint8"),
+                    "label": datasets.features.ClassLabel(names=["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]),
```
Comment (lhoestq) [inline datasets/cifar10/cifar10.py:63]: Can you use the actual class names here ? (airplane, automobile, etc.)

File: datasets/cifar10/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+
```
Comment (lhoestq) [inline datasets/cifar10/README.md:1]:
```suggestion
---
annotations_creators:
- crowdsourced
language_creators:
- found
languages: []
licenses:
- unknown
multilinguality: []
size_categories:
- n<1K
source_datasets:
- extended|other-80-Million-Tiny-Images
task_categories:
- other
task_ids:
- other-other-image-classification
---

```

**Review by lhoestq — APPROVED** [review]
Nice thank you !

**Comment by SBrandeis** [comment]
Yee a Computer Vision dataset!

**Comment by lhoestq** [comment]
Yep, the first one ! Thank @czabo

**Pull request merged**